R/attributes.R

```diff
@@ -22,7 +24,12 @@ check_attributes <- function(x,
                              class = TRUE,
                              x_name = substitute(x),
                              error = TRUE) {
-  lifecycle::deprecate_soft("0.5.1", "check_attributes()", NULL)
+  lifecycle::deprecate_soft(
+    "0.5.1", 
+    "check_attributes()", 
+    NULL,
+    "Pull out attributes with `attr()` and then use appropriate chk function"
```

Comment:
change details to "Pull out attributes with `attributes()` and then use appropriate chk function"

Reply:
done